Velocity_Model/z.py

```diff
@@ -205,7 +122,7 @@ def calculate_z_sigma(stat_df, version):
     """
     sigma_df = pd.DataFrame(index=stat_df.index)
     sigma_df["in_basin_flag"] = False
-    for outline_fp in basin_outlines[version]:
```

Comment:
What's the reason you renamed this variable? I think outlines make it a bit clearer.

Reply:
in get_z.py, this dictionary is imported as "vm_versions". 
The original name `basin_outlines` goes like:
```
from Velocity_Model.z import basin_outlines as vm_versions
```
then

```
 21     parser.add_argument(
 22         "-v",
 23         "--version",
 24         help="VM version to use",
 25         default="2.07",
 26         choices=vm_versions.keys(),
 27     )
```
I found it intuitively easier if it is explicitly named something-dict, to indicate `keys()` is doing something valid.
(come to think of it, I could have removed `as vm_version` too, but I didn't touch that out of respect for the original auther)

Besides, this dictionary contains a list of files per version, not the outline itself. Renaming this variable gives that subtle separation.

In gen_basin_stats.py,
```
basin_files = basin_dict[BASIN_VER]
basin_fp = basin_files[last_basin_idx]
basin = np.loadtxt(basin_fp)
basin_outline = mpltPath.Path(basin)
```
This reads better than what it could have been with the original `basin_outlines`
```
basin_files = basin_outlines[BASIN_VER]
basin_fp = basin_files[last_basin_idx]
basin = np.loadtxt(basin_fp)
basin_outline = mpltPath.Path(basin)
```
As the returned object from mpltPath.Path() is finally what I would happy call `basin_outline`.
If I retained the original variable name, the plural of `basin_outline` is a dictionary, while the singular is an actual basin_outline, which felt a bit weird.

Reply:
Dict and list were renamed in the latest commit. @jasonmotha